packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Added
+- SF-1073: Created new actions `selectMutlipleRefinements` and `selectMutliplePastPurchaseRefinements` actions.
+  - Both actions take in navigationIds and an array of indices when creating associated actions.
+
```

Comment:
Changelog entries should describe use cases, not implementations/call signatures. Consider describing *what* these do, not what they require in order to function.

Reply:
Noted, updated to reflect suggestion.